components/BannerNotificationContainer.tsx

```diff
@@ -60,16 +66,31 @@ export const BannerNotificationContainer: React.FC = () => {
 
       {bannerNotificationController.isBiometricUnlock && (
         <BannerNotification
-          type="success"
+          type={BannerStatusType.SUCCESS}
           message={t('alternateBiometricSuccess')}
           onClosePress={bannerNotificationController.DISMISS}
           testId={'alternateBiometricSuccess'}
           key={'updateBiometric'}
         />
       )}
+
+      {verificationStatus != null && showBannerNotificationContainer && (
+        <BannerNotification
+          type={verificationStatus.statusType}
+          message={t(`VcVerificationBanner:${verificationStatus?.statusType}`, {
+            vcDetails: `${t(`VcDetails:${verificationStatus.vcType}`)} ${
+              verificationStatus.vcNumber
+            }`,
+          })}
+          onClosePress={() => controller.RESET_VERIFICATION_STATUS()}
```

Comment:
There is no reason behind this
first i have called RESET_VERIFICATION_STATUS event with parenthesis so had to pass this as a arrow function to the onClosePress so that it wont be called immediately when the component is rendered and until we press on it. I will remove this to keep it consistent everywhere

Reply:
Done